changelogs/fragments/fix_ospfv2_integration_test.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

this is test fix , trivial should be fine 
```suggestion
trivial:
```